docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
I think probably the best place to do it would be in the markdown2rst filter:
https://github.com/jupyter/nbconvert/blob/ca9094c375dfab7adf694d1f6405ea7e17937c43/nbconvert/filters/markdown.py#L71